launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -214,7 +214,14 @@ QString MinecraftInstance::binRoot() const
 QString MinecraftInstance::getNativePath() const
 {
 #ifdef Q_OS_FREEBSD
-    QDir natives_dir("/usr/local/lib/lwjgl/");
+    if(instance.getPackProfile()->getComponentVersion("net.minecraft") >= 1.13)
```

Comment:
I don't think that will work. Look for the presence of LWJGL3 instead.
```suggestion
    if(!instance.getPackProfile()->getComponentVersion("org.lwjgl3").isEmpty())
```

Reply:
Why not

if(instance.getPackProfile()->getComponentVersion("org.lwjgl").isEmpty())